### PR TITLE
 fix completions for string literal types with template string (#19162)

### LINKED ISSUE #19162: No suggestions for string literal types when using an empty template string
From https://github.com/Microsoft/vscode/issues/36161

**TypeScript Version:**  2.6.0-20171013

**Code**

```ts
let count: 'one' | 'two';
count = `|`
```

Request auto complete at the `|`

**Expected behavior:**
Returns entries for `'one'` and `'two'`

**Actual behavior:**
No entries returned. 

We see the correct behavior if you use single or double quotes:

screen shot 2017-10-13 at 10 17 27 am

### PR DESCRIPTION
**Pull request by Kingwl**
Fixes #19162

### CODE REVIEW

File: src/services/completions.ts [hunk 1]
```diff
@@ -194,15 +194,15 @@ namespace ts.Completions {
             return PathCompletions.getStringLiteralCompletionEntriesFromModuleNames(<StringLiteral>node, compilerOptions, host, typeChecker);
         }
         else if (isEqualityExpression(node.parent)) {
+            // Get completions from the type of the switch expression
```
Comment (DanielRosenwasser) [inline src/services/completions.ts]: Why is the comment here swapped around?

**Comment by mhegazy** [comment]
@Kingwl  can you fix the comments please?

**Comment by Kingwl** [comment]
@mhegazy fixed

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**